scripts/docs/json_builder.py

```diff
@@ -0,0 +1,223 @@
+import json
+import os
+
+import pkgutil
+import importlib
+import sys
+from importlib.machinery import ModuleSpec
+from types import ModuleType, FunctionType
+import inspect
+from typing import Dict
+import re
+
+import docstring_parser
+from docstring_parser import parse
+from docstring_parser.common import DocstringStyle
+
+LOG = open("log.txt", "w+")
+
+
+def docstring_fixes(docstr: str) -> str:
+    docstr = re.sub(r"Args:[\n ]*None", "", docstr)  # Fix for Args: None in docstrings, which is not valid
```

Comment:
Make global variable with `re.compile` and use it instead of `re.sub(r"Args:[\n ]*None", "", docstr)`